test/test_transformers.py

```diff
@@ -1208,7 +1208,7 @@ class TestSDPAFailureModes(NNTestCase):
     @onlyCUDA
     @unittest.skipIf(not PLATFORM_SUPPORTS_FUSED_SDPA or not isSM86or89Device,
```

Comment:
I have yet to update `PLATFORM_SUPPORTS_FUSED_SDPA` to exclude windows. I am surprised CI/CD is green, and think this indicates a lack of coverage for sm80 + on windows. But regardless I need audit uses of this, and maybe make a PLATFORM_SUPPORTS_FLASH_SDPA and PLATFORM_SUPPORTS_MEM_EFF_SDPA